app/models/scc_account.rb

```diff
@@ -230,4 +254,32 @@ def update_scc_products(upstream_products)
       end
     end
   end
+
+  # validate the subscription status of a product/repo
+  # no saving to database
+  # params: elements: scc repos or products, Array or ActiveRecord_(*)
+  #         scc_id: scc_id of the element that should be revalidated
+  # return: elements where for the element with scc_id subscription_valid is true
+  def revalidate_subscription_status(elements, scc_id)
+    return nil if elements.nil?
+
+    revalidate = elements.find { |e| e.scc_id == scc_id }
+    revalidate.subscription_valid = true unless revalidate.nil?
+    # return modified list
+    elements
+  end
+
+  # set all products/repos invalid
+  # params: items_to_invalidate: ActiveRecord_(*)
+  #         save_record: store in database or not (default)
+  # return: ActiveRecord elements with invalidated subscription status
+  def invalidate_subscription_status(items_to_invalidate, save_record = false)
+    unless items_to_invalidate.empty?
```

Comment:
Rubocop change it, I think, it's a fair:
`scc_manager/app/models/scc_account.rb:277:5: C: [Corrected] Rails/Present: Use if items_to_invalidate.present? instead of unless items_to_invalidate.blank?.`

Reply:
Even better :+1: